### ISSUE TITLE
handle_grow_and_garden slowly

### ISSUE THREAD

**Issue by ljj0522** [Zlkcyber/hanafuda#3]
need to speed up this post rate. cant reach 300 times every hours.

**Comment by Zlkcyber** [comment]
yup the api url very slowly, already update the script to try make batch request

**Comment by ljj0522** [comment]
Error: Unexpected response format: {'errors': [{'message': 'Unauthorized', 'extensions': {'code': 'UNAUTHORIZED'}}]}
 is maybe change account dont change headers['authorization']
